### PR TITLE
Quickfort set

### PR DESCRIPTION
**Pull request by myk002**
DFHack/dfhack#499 Implement the quickfort set command and reading from the dfhack-config/quickfort/quickfort.txt config file

### CODE REVIEW

File: quickfort.lua [hunk 1]
```diff
@@ -151,6 +151,26 @@ For more info, see: https://docs.dfhack.org/en/stable/docs/_auto/base.html#quick
 ]]
 end
 
+local function set_setting(key, value)
+    if quickfort_common.settings[key] == nil then
+        error(string.format('error: invalid setting: "%s"', key))
```
Comment (lethosor) [inline quickfort.lua:156]: You could probably use [qerror](https://docs.dfhack.org/en/stable/docs/Lua%20API.html#exception-handling) for user-facing error messages like these

**Comment by myk002** [comment]
yes, just setting them in memory. I note in the help text that changes made with quickfort set only apply to the current session and are not persistent.

File: quickfort.lua (hunk 1 above)
Comment (myk002) [reply quickfort.lua]: thanks! done.

**Comment by lethosor** [comment]
The help text was already present from an older PR, so I missed it :)

**Pull request merged**